src/Core/Components/InputFile/FluentInputFile.razor.js

```diff
@@ -21,3 +21,51 @@ export function previewImage(inputElem, index, imgElem) {
     imgElem.src = url;
     imgElem.alt = inputElem.files[index].name;
 }
+
+export function initializeFileDropZone(containerElement, inputFile) {
+    function onDragHover(e) {
+        e.preventDefault();
+        containerElement.setAttribute("drop-files", "true");
+    }
+
+    function onDragLeave(e) {
+        e.preventDefault();
+        containerElement.removeAttribute("drop-files");
+    }
+
+    // Handle the paste and drop events
+    function onDrop(e) {
+        e.preventDefault();
+        containerElement.removeAttribute("drop-files");
+
+        // Set the files property of the input element and raise the change event
+        inputFile.files = e.dataTransfer.files;
+        const event = new Event('change', { bubbles: true });
+        inputFile.dispatchEvent(event);
+    }
+
+    function onPaste(e) {
+        // Set the files property of the input element and raise the change event
+        inputFile.files = e.clipboardData.files;
+        const event = new Event('change', { bubbles: true });
+        inputFile.dispatchEvent(event);
+    }
+
+    // Register all events
+    containerElement.addEventListener("dragenter", onDragHover);
+    containerElement.addEventListener("dragover", onDragHover);
+    containerElement.addEventListener("dragleave", onDragLeave);
+    containerElement.addEventListener("drop", onDrop);
+    containerElement.addEventListener('paste', onPaste);
```

Comment:
I don't understand how this ‘paste’ function can work with our component.
1. Because, by default, when the user clicks on the `FluentInputFile`, the `Open File` dialog box opens and the user cannot paste the file.
2. This functionality must be removed when the component is deactivated (see the last example in the demo site).